src/background/controllers/accountController.ts

```diff
@@ -127,6 +127,8 @@ export default class AccountController extends IController {
   * @param mnemonic The mnemonic to derive the wallet from.
   */
   public importMnemonic = async (accountName: string, mnemonic: string) => {
+    if (mnemonic === '' || mnemonic === null || mnemonic === undefined) { throw Error('invalid mnemonic'); }
```

Comment:
can we put the code inside the {} on a new line? (same comment for the other two places).

unless you want to inline the logic without the {}

Reply:
`!!mnemonic` should handle all three of these cases